### PR TITLE
Editing formatting / consistency

### PR DESCRIPTION
**Pull request by christinerose**
Changed opam to Opam in the previously approved file (ReadMe, Overview, and Quickstart)
Edited usage.rst and dune-files.rst for formatting, consistency, and syntax flow

### CODE REVIEW

File: doc/dune-files.rst [hunk 1]
```diff
@@ -433,155 +432,155 @@ format of library stanzas is as follows:
...
+  name it's part of and optionally followed by a dot, then anything else you
+  want. The package name must also be one of the packages that Dune knows about,
   as determined by the :ref:`opam-files`
 
-- ``(package <package>)`` Install private library under the specified package.
+- ``(package <package>)`` installs a private library under the specified package.
   Such a library is now usable by public libraries defined in the same project.
-  The findlib name for this library will be ``<package>.__private__.<name>``,
-  however the library's interface will be hidden from consumers outside the
+  The ``findlib`` name for this library will be ``<package>.__private__.<name>``;
```
[the remark this answers is not in the document]
Comment (christinerose) [reply doc/dune-files.rst:460]: That's even better capitalizing it.

File: doc/quick-start.rst [hunk 2]
```diff
@@ -89,7 +112,7 @@ And build it with:
 
 The executable will be built as ``_build/default/hello_world.exe``
 
-Defining a library using Lwt and ocaml-re
+Defining a Library Using Lwt and ``ocaml-re``
```
[the remark this answers is not in the document]
Comment (christinerose) [reply doc/quick-start.rst:115]: Understood.

File: doc/dune-files.rst [hunk 3]
```diff
@@ -18,35 +18,35 @@ Additionally, they can contains the following stanzas.
 
 .. _using:
 
-using
+Using
```
[the remark this answers is not in the document]
Comment (christinerose) [reply doc/dune-files.rst:21]: That sounds good.

File: doc/dune-files.rst [hunk 4]
```diff
@@ -1911,32 +1911,32 @@ Where ``<optional-fields>`` are:
...
 
-Upgrading from version 0.1
+Upgrading from Version 0.1
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- The 0.2 version of the stanza requires at least mdx 1.9.0. If you encounter
+- The 0.2 version of the stanza requires at least MDX 1.9.0. If you encounter
   an error such as, ``ocaml-mdx: unknown command `dune-gen'``, then you
-  should upgrade mdx.  
+  should upgrade MDX.
```
[the remark this answers is not in the document]
Comment (christinerose) [reply doc/dune-files.rst:1923]: The Comms team have been encouraging the capitalization of proper names, but I'll definitely check with them to be sure.

File: doc/dune-files.rst [hunk 5]
```diff
@@ -264,70 +264,70 @@ following field to the ``dune-project`` file:
...
+``dune build``, ``dune runtest``, or something else), the Opam files get
 generated.
 
 Some or all of these fields may be overridden for each package of the project, see
 :ref:`package`.
 
 .. _package:
 
-package
+Package
```
Comment (rgrinberg) [inline doc/dune-files.rst:293]: Ditto here and other subtitles. They all refer to concrete snippets that must appear in dune files in lower case form (sometimes with underscores).

Comment (christinerose) [reply doc/dune-files.rst:293]: Understood! That’s fine if that’s the way it needs to be for accuracy and clarity. :)

**Comment by rgrinberg** [comment]
@christinerose is this ready for review?

**Comment by christinerose** [comment]
Yes. I believe so!

**Comment by rgrinberg** [comment]
Merged after tweaking the commit messages a little.

**Pull request closed without merging**